### PR TITLE
Daria

### PR DESCRIPTION
**Pull request by DariaSavchenko12**

### CODE REVIEW

**Comment by DariaSavchenko12** [comment]
week2 (first from the bottom) is a correct link

**Comment by feyzanurdogan** [comment]
excellent

**Comment by DariaSavchenko12** [comment]
I will fix it something

**Review by feyzanurdogan — APPROVED** [review]

**Comment by feyzanurdogan** [comment]
> I will fix it something

Oh okay. should i close it?

**Comment by DariaSavchenko12** [comment]
Did you see week2 task?

**Comment by feyzanurdogan** [comment]
> Did you see week2 task?

Yes i saw. There are 3 tasks in week3 package

**Comment by DariaSavchenko12** [comment]
Oh good, thats is should be 3 tasks

**Review by robertiele1040 — APPROVED** [review]

**Review by robertiele1040 — APPROVED** [review]

**Review by Anguz2 — COMMENTED** [review]
Looks good

File: src/Daria/week2/SwapNumbers.java
Comment (Anguz2) [inline src/Daria/week2/SwapNumbers.java:1]: It looks good to me but I can't see where to approve it

File: src/Daria/week2/Print_Consecutive_Numbers.java
Comment (Anguz2) [inline src/Daria/week2/Print_Consecutive_Numbers.java:1]: Looks good

**Pull request merged**